sympy/stats/crv.py

```diff
@@ -181,7 +182,9 @@ def _inverse_cdf_expression(self):
         x, z = symbols('x, z', real=True, positive=True, cls=Dummy)
         # Invert CDF
         try:
-            inverse_cdf = solve(self.cdf(x) - z, x)
+            inverse_cdf = solveset(self.cdf(x) - z, x)
+            if isinstance(inverse_cdf, Intersection) and inverse_cdf.args[0] == S.Reals:
```

Comment:
Doing this raises 
>SymPyDeprecationWarning: 
    is_real has been deprecated since SymPy 0.7.6. Use is_subset(S.Reals)

and the test fails.